components/Spaces/Chat/index.js

```diff
@@ -6,17 +6,23 @@ import SendIcon from '@material-ui/icons/Send';
 import Conversation from './Conversation';
 import { useConversation } from '../../../context/ConversationProvider';
 
-const Chat = () => {
-  const [text, setText] = useState('');
-  const [error, setError] = useState(false);
-  const { sendMessage } = useConversation();
+const Chat = ({ conversation, setConversation, peersRef }) => {
+  const [text, setText] = useState('')
+  const [error, setError] = useState(false)
+  const { sendMessage } = useConversation()
 
   const submitHandler = (e) => {
-    e.preventDefault();
+    e.preventDefault()
+    setConversation(prevConversation => {
+      return [...prevConversation, { text: text, fromMe: true }]
+    })
+    peersRef.current.forEach((peer, i) => {
+      if (peer) {
+        peer.peer.send(text)
```

Comment:
why are there two peers? I get that peersRef.current is an array of peers, but shouldn't it just be `peer.send(<text>)`?

Reply:
Nevermind I think I found where in the code you did that. Can you rename the first peer to `peerObj` for clarity?